requirements/deploy.txt

```diff
@@ -171,7 +173,7 @@ djangorestframework==3.14.0
     # via
     #   -r requirements/pip.txt
     #   drf-extensions
-djangorestframework-api-key==2.3.0
+djangorestframework-api-key==3.0.0
```

Comment:
I think this is the version that comes with the CPU improvement on the key hash 💯